data/lavezzo2020suppression/lavezzo2020suppression.yaml

```diff
@@ -0,0 +1,2196 @@
+# yaml-language-server: $schema=../.schema.yaml
+title: Suppression of a SARS-CoV-2 outbreak in the Italian municipality of Vo'
+doi: 10.1038/s41586-020-2488-1
+description: >
+  This study was conducted in the Italian municipality of Vo. A lockdown was implemented
+  after the first death of pneumonia was reported. Two rounds of surveys and virus
+  tests were conducted with the first survey near the start of the lockdown and the
+  second one at the end of the lockdown
+analytes:
+  RdRp_first_pos:
+    description: >
+      SARS-CoV-2 RNA genome copy concentration calculated from evaluation of RdRp
+      gene, and the reference event is the first positive test day (confirmation date).
+    specimen: oropharyngeal_swab
+    biomarker: SARS-CoV-2
+    gene_target: RdRp
+    limit_of_quantification: unknown
+    limit_of_detection: unknown
+    unit: gc/mL
+    reference_event: confirmation date
+  E_first_pos:
+    description: >
+      SARS-CoV-2 RNA genome copy concentration calculated from evaluation of E gene,
+      and the reference event is the first positive test day (confirmation date).
+    specimen: oropharyngeal_swab
+    biomarker: SARS-CoV-2
+    gene_target: E
+    limit_of_quantification: unknown
+    limit_of_detection: unknown
+    unit: gc/mL
+    reference_event: confirmation date
+  RdRp_symptom:
+    description: >
+      SARS-CoV-2 RNA genome copy concentration calculated from evaluation of RdRp
+      gene, and the reference event is symptom onset.
+    specimen: oropharyngeal_swab
+    biomarker: SARS-CoV-2
+    gene_target: RdRp
+    limit_of_quantification: unknown
+    limit_of_detection: unknown
+    unit: gc/mL
+    reference_event: symptom onset
+  E_symptom:
+    description: >
+      SARS-CoV-2 RNA genome copy concentration calculated from evaluation of E gene,
+      and the reference event is symptom onset.
+    specimen: oropharyngeal_swab
+    biomarker: SARS-CoV-2
+    gene_target: E
+    limit_of_quantification: unknown
+    limit_of_detection: unknown
+    unit: gc/mL
+    reference_event: symptom onset
+participants:
+- attributes:
+    age: 65
+    sex: male
+  measurements:
+  - analyte: RdRp_first_pos
+    time: 0.0
+    value: 116.8530752
+  - analyte: RdRp_first_pos
+    time: 8.0
+    value: negative
+  - analyte: E_first_pos
+    time: 0.0
+    value: 2806.098051
+  - analyte: E_first_pos
+    time: 8.0
+    value: negative
+- attributes:
```

Comment:
The data were double-entered. The numbers in this one are the same as the last person.

Reply:
This bug has been addressed.